books/RayTracingTheNextWeek.html

```diff
@@ -1215,7 +1215,7 @@
                 vec3 outward_normal = (rec.p - center) / radius;
                 rec.set_face_normal(r, outward_normal);
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-                get_sphere_uv((rec.p-center)/radius, rec.u, rec.v);
+                get_sphere_uv(outward_normal, rec.u, rec.v);
```

Comment:
This has some interesting implications.
The functionality is identical before and after change.

A texture shaded on the inside is reversed. Which is what you'd expect.